build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id 'java'
+    id 'jacoco'
+    id "nebula.lint" version "17.3.0"
+}
+
+jacoco {
+    toolVersion = "0.8.5"
+}
+
+group 'org.example'
```

Comment:
It would be nice to have not `org.example`, but a bit more suitable for project group.